doc/getting-started/understanding-config-files.md

```diff
@@ -21,6 +21,64 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 Your __block-producing__ node must __ONLY__ talk to your __relay nodes__, and the relay node should talk to other relay nodes in the network. Go to our telegram channel to find out IP addresses and ports of peers.
 
+#### The P2P topology.json file
+
+Tells your node to which nodes in the network it should talk to. A minimal version of this file looks like this:
+
+```json
+{
+  "LocalRoots": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "addr": "x.x.x.x",
+              "port": 3001
+            }
+          ],
+          "advertise": false
+        },
+        "valency": 1
+      }
+    ]
+  },
+  "PublicRoots": [
+    {
+      "publicRoots" : {
+        "addrs": [
+          {
+            "addr": "y.y.y.y",
+            "port": 3002
+          }
+        ],
+        "advertise": false
+      }
+    }
+  ],
+  "useLedgerAfterSlot": 0
+}
+```
+
+* The main difference between `LocalRoots` and `PublicRoots` is that with the former
```

Comment:
Why not "LocalPeers" and "PublicPeers"? Roots seems a bit opaque to me.

Reply:
These are not just peers, these are indeed the roots of the network graph.  That's consistent with our naming convention in the code base (and documentation, though that's mostly haddock right now).

Reply:
Can we just clarify that root essentially means initial?

Reply:
Yes, I'll do it.